### PR TITLE
Updating README.md for supported openSUSE version and updating link to OS versions supported by CoreFx

### PR DESCRIPTION
**Pull request by stknohg**
## PR Summary

* Update openSUSE 42.2 to 42.3, and adding [openSUSE Leap 15](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6#installation---opensuse-leap-15).
    * [Installation - openSUSE Leap 15](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6#installation---opensuse-leap-15)
* Update CoreFx version 2.0 to 2.1.
    * [.NET Core 2.1 - Supported OS versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md)

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by RDIL — APPROVED** [review]

**Review by anmenaga — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,20 +25,20 @@ If you are new to PowerShell and would like to learn more, we recommend reviewin
...
+| [Windows (x86)][corefx-win]                     | [.msi][rl-windows-86]   | [.msi][pv-windows-86] | [Instructions][in-windows]    |
+| [Ubuntu 18.04][corefx-linux]                    | [.deb][rl-ubuntu18]     | [.deb][pv-ubuntu18]   | [Instructions][in-ubuntu18]   |
+| [Ubuntu 16.04][corefx-linux]                    | [.deb][rl-ubuntu16]     | [.deb][pv-ubuntu16]   | [Instructions][in-ubuntu16]   |
+| [Ubuntu 14.04][corefx-linux]                    | [.deb][rl-ubuntu14]     | [.deb][pv-ubuntu14]   | [Instructions][in-ubuntu14]   |
+| [Debian 9][corefx-linux]                        | [.deb][rl-debian9]      | [.deb][pv-debian9]    | [Instructions][in-deb9]       |
+| [CentOS 7][corefx-linux]                        | [.rpm][rl-centos]       | [.rpm][pv-centos]     | [Instructions][in-centos]     |
+| [Red Hat Enterprise Linux 7][corefx-linux]      | [.rpm][rl-centos]       | [.rpm][pv-centos]     | [Instructions][in-rhel7]      |
+| [openSUSE 42.3, openSUSE Leap 15][corefx-linux] | [.rpm][rl-centos]       | [.rpm][pv-centos]     | [Instructions][in-opensuse]   |
```
Comment (TravisEz13) [inline README.md]: Please don't include openSUSE Leap in the list of supported platforms

https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md

File: README.md [hunk 2]
```diff
@@ -25,20 +25,20 @@ If you are new to PowerShell and would like to learn more, we recommend reviewin
...
+| [Fedora 27, Fedora 28][corefx-linux]            | [.rpm][rl-centos]       | [.rpm][pv-centos]     | [Instructions][in-fedora]     |
+| [macOS 10.12+][corefx-macos]                    | [.pkg][rl-macos]        | [.pkg][pv-macos]      | [Instructions][in-macos]      |
```
Comment (TravisEz13) [inline README.md]: Can you updated this to 10.13+ ?

Comment (TravisEz13) [reply README.md]: not blocking

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stknohg) [reply README.md]: ok.I will remove 'openSUSE Leap 15'.

File: README.md (hunk 2 above)
Comment (stknohg) [reply README.md]: Did the support target of PowerShell Core 6.1 change?

Comment (TravisEz13) [reply README.md]: Yes, we can no longer build or test 10.12, except manually.  So, we cannot guarantee compatibility.  This is particularly a problem for features that rely on native components which we build.  Most other features will likely work without issue.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**